tests/gumjs/script.c

```diff
@@ -884,6 +885,15 @@ SCRIPT_TESTCASE (native_pointer_to_match_pattern)
 #endif
 }
 
+SCRIPT_TESTCASE (native_pointer_can_be_constructed_from_64bit_value)
+{
+  COMPILE_AND_LOAD_SCRIPT (
+      "send(ptr(0x1FFFFFFFF).equals(ptr(uint64(0x1FFFFFFFF))));"
+      "send(ptr(0x2FFFFFFFF).equals(ptr(int64(0x2FFFFFFFF))));");
```

Comment:
These two should be swapped - we should check them in the same order as their chronological listing: signed then unsigned.